packages/kit/src/runtime/server/cookie.js

```diff
@@ -113,6 +117,16 @@ export function get_cookies(request, url, trailing_slash) {
 		 * @param {import('./page/types.js').Cookie['options']} options
 		 */
 		set(name, value, options) {
+			// TODO: remove this check in 3.0
+			if (DEV) {
```

Comment:
I think I might show this warning in prod too. It's possible that users have different cookie names in dev and prod and we'd still want them to be aware if it's only happening in prod

Reply:
I've removed the DEV check